module-federation/next.js
# Webpack 5 (Active upgrade work)

This is a diff for Zeit to look at and also know what branch to pull down - not to be merged at this point in time due to broken HMR, and broken dynamic routing

# How to test issue

Use any of your examples after `yarn install`ing the project. Make sure examples are linked (if need be) so that it's using the next.js in the monorepo, not pulling a new one from NPM

use `examples/dynamic-routing` and run `yarn build`

### Main Issues requiring support
- HMR doesn't work correctly. I sent a video of the different solutions I attempted.  
- The following error is thrown during builds `Error occurred prerendering page "/post/[id]". `

<img width="937" alt="Screen Shot 2020-03-16 at 6 48 19 PM" src="https://user-images.githubusercontent.com/25274700/76814045-f0ce9280-67b6-11ea-9754-344797bf6a39.png">
